docs/src/development/public-ips.md

```diff
@@ -1,254 +1,46 @@
 ---
+# Data: Add and edit regions in `docs/data/regions.yaml`.
+# Styles: See `docs/static/styles/user-customizations.css`.
+# Table shortcode: See `docs/layouts/shortcodes/region_ips.html`.
 title: "Public IP addresses"
 weight: 14
 sidebarTitle: "Public IPs"
 description: |
   Platform.sh regions reach the outside through a limited number of IP addresses.
+  These IP addresses are stable, but not guaranteed to never change.
+  Prior to any future change, all affected customers will receive ample warning.
 ---
 
 {{< description >}}
 
-{{< note >}}
+## Glossary
 
-These IP addresses are stable, but not guaranteed to never change.
-Prior to any future change, all affected customers will receive ample warning.
+**Data Location Guarantee:** Customer data never leaves the region. 
```

Comment:
I feel who the customer is might be slightly ambiguous. Like if it's an ecommerce site or something -- is it site data or the data of their customers or something else. I also feel like the original issue showed that it wasn't clear with log data, so maybe that should be explicitly added?